gatsby-node.js

```diff
@@ -243,13 +243,13 @@ exports.sourceNodes = ({ actions, createNodeId, createContentDigest }) => {
 	const today = dayjs().hour(0).minute(0).second(0);
 	for (const rawEvent of events) {
 		promises.push(actions.createNode({
-			// if conferenceLink does not exist in any of the rawEvents
+			// if rsvpLink does not exist in any of the rawEvents
 			// then its value will default to empty, otherwise it will be
 			// overwritten by the actual value
 			// See https://github.com/gatsbyjs/gatsby/issues/6800
```

Comment:
Looks like this issue has been resolved so in the next PR we should implement the fix